Paybox/System/Base/Request.php

```diff
@@ -147,10 +148,23 @@ public function getForm($options = array())
         $options['csrf_protection'] = false;
 
         $parameters = $this->getParameters();
-        $builder = $this->factory->createNamedBuilder('', 'form', $parameters, $options);
-
-        foreach ($parameters as $key => $value) {
-            $builder->add($key, 'hidden');
+        // If symfony version is 3.* then we use the FQCN for form types
+        // Else we use the IDs.
+        if (substr(Kernel::VERSION, 0, 1) === "3") {
```

Comment:
BTW, `substr` is a big and painful workaround see [`version_compare`](http://php.net/manual/fr/function.version-compare.php)